app/src/main/java/org/fossasia/susi/ai/login/LoginPresenter.kt

```diff
@@ -125,6 +127,28 @@ class LoginPresenter(loginActivity: LoginActivity): ILoginPresenter, ILoginModel
         }
     }
 
+    override fun onSuccessSetting(response: Response<UserSetting>) {
+
+        loginView?.showProgress(false)
```

Comment:
No, in that case user won't be logged in but the progress dialog will dismiss. Nevermind, leave it. I thought something else. It's fine. Sorry, for trouble.